cookbook/configuration/configuration_organization.rst

```diff
@@ -34,7 +34,7 @@ default Symfony Standard Edition follow this structure:
 
 .. code-block:: text
 
-    <your-project>/
+    your-project/
```

Comment:
Why isn't the `bin` directory present in here ? Is it expected ?
It has nothing to do with configuration files, but same as for new `var` and `tests` directories, right ? (Well, eventually `var` for environments)